src/components/sbb-toggle-check/sbb-toggle-check.tsx

```diff
@@ -27,10 +27,10 @@ export class SbbToggleCheck implements AccessibilityProperties {
   @Prop() public icon = 'tick-small';
 
   /** The disabled prop for the disabled state. */
-  @Prop() public disabled!: boolean;
+  @Prop({ reflect: true }) public disabled = false;
 
   /** The required prop for the required state. */
-  @Prop() public required?: boolean;
+  @Prop({ reflect: true }) public required = false;
```

Comment:
If there's no style difference (see question 3 here: https://github.com/lyne-design-system/lyne-components/pull/1294#issuecomment-1310585742) then it could be removed

Reply:
Sorry: I was sure it was on the checkbox-group, I just realized it's on the toogle! 

So answer is: I don't know :D

Reply:
Removed and resolved as done in the checkbox.